dhis-2/dhis-web-api/src/main/java/org/hisp/dhis/webapi/controller/tracker/export/event/EventRequestParamsMapper.java

```diff
@@ -97,6 +99,8 @@ public EventOperationParams map(RequestParams requestParams) throws BadRequestEx
 
     validateUpdateDurationParams(requestParams);
 
+    validateOrgUnitParams(requestParams.getOrgUnit(), orgUnitMode);
```

Comment:
Maybe not part of this PR but should we make `orgUnit` field behave as the other endpoints.
`Event` is the only one that accepts one `orgUnit`, does it make sense?

Reply:
Very good point! Not sure how much work it would be to allow that. It would at least be a backwards compatible change

Reply:
I've seen that `trackedEntities` and `enrollments` accept a list of org units uids and event expects a single uid. I don't know the reason behind though, so I can't tell if it makes sense. What do you think? Should we refactor it to accept a list of uids too?

Reply:
I am in favor of all endpoints accepting a list in `orgUnits` request parameter but I don't know either if it makes sense to implement that for events. 🤷🏻

Reply:
Me too, @enricocolasante does it make sense from the user perspective to accept multiple org units for events?
In case it does, we can use the opportunity and edit the docs, because based on them, enrollments does not support a list of org units: https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html?h=enrollments+develop#enrollments-get-apitrackerenrollments

We should have it written as in tracked entities: https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html?h=master+develop#tracked-entities-get-apitrackertrackedentities

Reply:
OpenAPI spec is the source of truth from now on: https://github.com/dhis2/dhis2-core/blob/master/dhis-2/dhis-web-api/src/main/resources/openapi/EnrollmentsExportController.md#parameterenrollmentrequestparamsorgunits 

We need to work on automaticall deploying the OpenAPI spec and linking it in https://docs.dhis2.org/en/develop/using-the-api/dhis-core-version-master/tracker.html instead of the tables in there.